### PR TITLE
Column selector position was aligned

### PR DESCRIPTION
**Pull request by uday-appam**
Column selector position was aligned.
image

### CODE REVIEW

File: BootStrap/index.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h5 class="modal-title" id="exampleModalLabel">Bootstrap</h5>
             aria-haspopup="true" aria-expanded="false">
             <i class="fa fa-list-alt"></i>
   </button>
-          <div class="dropdown-menu" id="dropdown-list" aria-labelledby="dropdownMenuButton">
+          <div class="dropdown-menu dropdown-menu-right" id="dropdown-list" aria-labelledby="dropdownMenuButton">
```
Comment (prasanna03) [inline BootStrap/index.html]: dropdown-menu-right -- It seems this class is not being used anywhere.

File: BootStrap/css/style.less [hunk 2]
```diff
@@ -114,7 +114,8 @@ body {
 }
 
 #dropdown-list {
-  span {
+  right: 0;
```
Comment (prasanna03) [inline BootStrap/css/style.less]: Check if the positioning of column selector works fine in all resolutions.

Comment (uday-appam) [reply BootStrap/css/style.less]: It's working in all resolution

File: BootStrap/index.html (hunk 1 above)
Comment (uday-appam) [reply BootStrap/index.html]: That is bootstrap class.
I didn't get what exactly not being used.

Comment (siddhu-osm) [reply BootStrap/index.html]: Not being used means, you didn't use that class in your custom styles.

Comment (uday-appam) [reply BootStrap/index.html]: That is bootstrap class.

Comment (uday-appam) [reply BootStrap/index.html]: No nee to write custom css for that

**Pull request merged**